server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -437,9 +438,13 @@ const orderMutations = {
           order = await updatePaymentMethodForSubscription(req.remoteUser, order, newPaymentMethod);
         }
 
-        // Unpause contribution
-        if (order.status === OrderStatuses.PAUSED) {
```

Comment:
For Stripe, this code was not reached because at this point the order was marked as active already by https://github.com/opencollective/opencollective-api/blob/cbfc984efc008c2f36eb903b4c2d5dfec236cfd9/server/lib/subscriptions.ts#L85